lib/Crypt/JWT.pm

```diff
@@ -280,7 +280,7 @@ sub _payload_unzip {
 sub _payload_enc {
   my ($payload) = @_;
   if (ref($payload) =~ /^(HASH|ARRAY)$/) {
-    $payload = encode_json($payload);
+    $payload = JSON->new->canonical(1)->encode($payload);
```

Comment:
it should be IMO `JSON->new->utf8->canonical->encode($payload);`

Reply:
Good call.  Thanks.